### ISSUE TITLE
Ugly website title

### ISSUE THREAD

**Issue by VGPReys** [haddocking/haddocking.github.io#659]
There an un-necessary default `Your awesome title` header at top left of every pages.

Same goes for the website `<title>` seen in tab

Can probably be configured from `_config.yml`.

Modifying `title: Your awesome title` to `title: Bonvin Lab` is the first step.

But I guess the full section should be edited.

Currently is:
```
title: Your awesome title
email: your-email@example.com
description: >- # this means to ignore newlines until "baseurl:"
  Write an awesome description for your new site here. You can edit this
  line in _config.yml. It will appear in your document head meta (for
  Google search results) and in your feed.xml site description.
baseurl: "" # the subpath of your site, e.g. /blog
url: "" # the base hostname & protocol for your site, e.g. http://example.com
twitter_username: jekyllrb
github_username:  jekyll
```

**Comment by amjjbonvin** [comment]
Oops 

I updated the Jekyll installation last weekend - must have messed up a few things…

**Comment by amjjbonvin** [comment]
Solved with commit #e37e2c5
